_posts/2018-01-01-futures-0.3.0-alpha.2.md

```diff
@@ -0,0 +1,61 @@
+
+The notable difference to before is that the term "task" no longer refers to the future that the task runs. Instead, from now on, the term "task" only refers to the concept of a logical thread, a kind of thread that doesn't directly correspond to a thread provided by the operating system. This concept is sometimes also referred to as "lightweight thread" or "green thread" - We, however, do not use these terms because they tend to cause confusion with real operating system threads.
+
+### `Try` impl for `Poll`
+
+The [`Poll`](https://doc.rust-lang.org/nightly/core/task/enum.Poll.html) enum now implements the [`Try`](https://doc.rust-lang.org/std/ops/trait.Try.html) trait. This means you now do this:
+
+```rust
+impl Future for MyFuture {
+    type Output = Result<T, E>;
+
+    fn poll(mut self: PinMut<Self>, cx: &mut task::Context, ) -> Poll<Result<T, E>> {
+        let pinned_child_future = self.child_future();
+        let ok_value = ready!(pinned_child_future.poll()?);
+        // ...
+    }
+}
+```
+
+- The `?`-operator used on a `Poll<Result<T, E>>` value:
+  - yields `Poll::Pending` from the expression if the value is not ready.
+  - yields `Poll::Ready(ok_value)` from the expression if the value is ready and successful.
+  - returns `Poll::Ready(Err(err_value))` from the function if the value is ready and an error.
+- The `ready!` macro used on a `Poll<T>` value:
+  - yields the value of type `T` from the expression if the value is ready.
+  - returns `Poll::pending` from the function if the value is not ready.
+
+Additionally `Try` was also implemented for `Poll<Option<Result<T, E>>>`, more details on that in the [pull request](https://github.com/rust-lang/rust/pull/52721).
+
+### Changes to futures-rs
+
+We revived the [changelog](https://github.com/rust-lang-nursery/futures-rs/blob/master/CHANGELOG.md)!
+
+No major changes happened to the futures crate since the last release. There were, however, numerous little changes. Take a look at the [changelog](https://github.com/rust-lang-nursery/futures-rs/blob/master/CHANGELOG.md) to get an overview.
+
+## What are we currently working on?
+
+### Compatiblity layer for futures 0.1
+
+We are currently working on an officially supported compatibility layer between `0.1` and `0.3`. This shim will make it possible to mix `0.1` futures with those from the 0.3 branch. The plan is to make it possible to use crates from the existing futures `0.1` ecosystem together with async functions and futures from the `0.3` crate. Additionally, this compatibility layer will also make it possible to gradually migrate exisiting applications and libraries to `0.3`. If you're curious, take a look the the [compatibility layer PR](https://github.com/rust-lang-nursery/futures-rs/pull/1119).
```

Comment:
Maybe link to the [WG-net issue](https://github.com/rust-lang-nursery/net-wg/issues/26) as well?